### DISCUSSION TITLE
How to install this in Linux?

### DISCUSSION THREAD

**Discussion by Syndicoon** [brackets-cont/brackets#101] [Q&A] [answered]
Hi everyone,

I use Linux Mint and the only version I have access to is version 1.x. 
Will there be a .deb or .rpm package uploaded? 
At the very least some directions on how to compile the source code would be nice. 
Right now I can't install any extensions. 

Thanks! I really love Brackets and would love to keep using it.

**Comment by KDurimel** [comment]
I tried to compile the source code but I failed too, it was a little tricky

+1 , some directions on how to compile or execute the source code would be nice

**Comment by abose** [comment, marked as answer]
TLDR: We are working on Linux support and plan to complete it by March
Update on the Linux situation. 
Closing thread as a duplicate.

https://github.com/brackets-cont/brackets/discussions/113
